BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/image_maker.sh

```diff
@@ -54,11 +53,16 @@ min_index=`cat temp_worst.csv`
 #make sure the correct python is loaded (default 2.7)
 module load python/3.7-2019.10
 module unload python/3.7-2019.10
-source $PSIMDIR/set_env.sh
+source /fs/ess/PAS1960/buildingPueoSim/set_env.sh
```

Comment:
Do we need to hard code the sourcing here? It looks like we're passing the directory variable $PSIMDIR to this script when we call it in Part_F_PUEO.sh.

Reply:
Good catch. Forgot to change that back after testing it.